test/engagement.e2e-spec.ts

```diff
@@ -1071,11 +1071,11 @@ describe('Engagement e2e', () => {
       ProjectStep.Rejected,
       EngagementStatus.Rejected,
     ],
-    [
-      ProjectStep.PendingTerminationApproval,
-      ProjectStep.Terminated,
-      EngagementStatus.Terminated,
-    ],
+    // [
```

Comment:
@CarsonF I commented out this element of the array which was causing the following error.  I'm not sure how to fix this, but putting my findings below:
```
 should update Engagement status to match Project step when it becomes PendingTerminationApproval
    [Unauthorized, Input, Client]: This transition is not available
```
at the following location: 
`at findTransition (src/components/workflow/workflow.service.ts:159:11)`